_webi/template.sh

```diff
@@ -181,7 +181,7 @@ function __bootstrap_webi() {
             # TODO wget -c --content-disposition "$my_url"
             set +e
             my_show_progress=""
-            if [[ $- == *i* ]]; then
+            if [ $- == *i* ]; then
```

Comment:
`if echo "$-" | grep -q 'i';` should work here

Reply:
I did find that there's a "simpler" way to do this (using only Posix Shell, not grep), but it's not as easy to read:
https://stackoverflow.com/a/73419937/151312